src/main/java/hudson/plugins/plot/Plot.java

```diff
@@ -451,23 +451,24 @@ private int getHeight() {
         return height;
     }
 
-    public Job<?, ?> getProject() {
+    public Job<?, ?> getJob() {
         return project;
     }
 
-    /**
-     * @see #setProject(Job)
-     */
-    public void setProject(AbstractProject<?, ?> project) {
-        setProject((Job) project);
+    public void setJob(Job<?, ?> job) {
+        this.project = job;
+    }
+
+    public AbstractProject<?, ?> getProject() {
+        return (AbstractProject<?, ?>) project;
```

Comment:
Probably want to mark as `@Deprecated`

Reply:
Also might do the instanceof check and return null as with the other one

Reply:
I thought about `null` but don't why didn't add it :) I'll do.